man/krahmer2018pcp.Rd

```diff
@@ -0,0 +1,36 @@
+\name{krahmer2018pcp}
+\alias{krahmer2018pcp}
+\docType{data}
+
+\title{
+  Subcellular Reorganization in Diet-Induced Hepatic Steatosis
+}
+
+\description{
+  
+  Data from 'Organellar Proteomics and Phospho-Proteomics Reveal Subcellular Reorganization in Diet-Induced Hepatic Steatosis' 
+  Developmental cell.
+  
+  Lipid metabolism is highly compartmentalized between cellular organelles that
+  dynamically adapt their compositions and interactions in response to metabolic
+  challenges. Here, we investigate how diet-induced hepatic lipid accumulation,
+  observed in non-alcoholic fatty liver disease (NAFLD), affects protein localization,
+  organelle organization, and protein phosphorylation in vivo. We develop a mass
+  spectrometric workflow for protein and phosphopeptide correlation profiling to
+  monitor levels and cellular distributions of 6,000 liver proteins and 16,000
+  phosphopeptides during development of steatosis. Several organelle contact
+  site proteins are targeted to lipid droplets (LDs) in steatotic liver,
+  tethering organelles orchestrating lipid metabolism. Proteins of the
+  secretory pathway dramatically redistribute, including the mis-localization
+  of the COPI complex and sequestration of the Golgi apparatus at LDs. This
+  correlates with reduced hepatic protein secretion. Our systematic in vivo
+  analysis of subcellular rearrangements and organelle-specific phosphorylation
+  reveals how nutrient overload leads to organellar reorganization and cellular dysfunction.
+  
+}
+
+\usage{
+  data("krahmer2018pcp")
+}
+
```

Comment:
Could you add *format*, *references* and *examples* sections. See `hyperLOPIT2015.Rd` for an example.

Reply:
added